### PR TITLE
Relativity.DataExchange.Client.SDK updated to version 1.15.30

### PR DESCRIPTION
**Pull request by dawidstoga-rel**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.0.0] - 2021-11-18
```
Comment (briantoms) [inline CHANGELOG.md:8]: This should probably be `5.12.0`. I don't see anything in here that would make a breaking change for consumers.

Comment (paulletang) [reply CHANGELOG.md:8]: ServiceFactory no longer can connect to the RSAPI.
That looks to me like it is a breaking change (but likely that it won't affect anybody).

**Comment by paulletang** [comment]
The workflow here is behaving differently that we've seen in the past for forks.
Previously it required someone on the team to approve running the build, and then it would use our secrets to run the tests.
Now it looks like it doesn't require that approval, and it uses the forked repository's secrets.

I think the difference is that the last PR was made by a user external to the org, and this one is internal to the org.

It's a problem that we'll need to figure out at some point, but for right now I'm going to make a new branch in our repo to test this out.
If the tests look good, I'll approve and merge this in if there are no other changes needed.

**Review by paulletang — APPROVED** [review]

**Comment by paulletang** [comment]
Hey @dawidstoga-rel I was running the tests for this PR, and it looks like one relating to document import is timing out.
image
https://github.com/relativitydev/relativity.testing.framework.api/runs/4258973723?check_suite_focus=true

Are there any changes between the two SDK versions that might be causing an issue here?

**Comment by dawidstoga-rel** [comment]
Relativity.DataExchange.Client.SDK need to be updated to 1.15.28 first (fix related to long retries)

**Comment by paulletang** [comment]
I ran the tests over [here](https://github.com/relativitydev/relativity.testing.framework.api/runs/4313683903?check_suite_focus=true)
Functionally this looks good, but we chatted in slack, and I would like the DataExchange.SDK package to pin to a real version of Relativity.Telemetry.DataContracts.Shared so that we don't have to globally ignore warnings for NU1603.

**Pull request merged**